app/assets/stylesheets/argo/argo.sass

```diff
@@ -189,3 +191,13 @@ button.close
 #bulk-spreadsheet-warning
   color: $color-cardinal-red
 
+.bulk-buttons
+  margin-left: 5px
+  margin-right: 5px
+
+.bulk-buttons-row
+  margin-top: 8px
+  margin-bottom: 8px
+
+.bulk-btn-danger
+  background-color: $color-cardinal-red
```

Comment:
Prefer `$brand-danger` here instead. Cardinal-red is a SU branding color, and shouldn't denote a dangerous item.